src/msgpack/CompiledLambdaActivatorFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq.Expressions;
+#if DNXCORE50
```

Comment:
It's better to supress warning of Resharper by comment. I dislike compilation directives